NEWS.md

```diff
@@ -1,3 +1,18 @@
+# messydates 0.4.0
+
+## Functions
+
+* Closed #72 by fixing issues with double unspecified components not being contracted correctly
+* Closed #73 by fixing bugs with the conversion of dates where month is spelled
+* Closed #74 by adding other logical comparison operators for 'mdate' objects (e.g. `<`, `>`, `<=`, `>=`)   (thanks @WerthPADOH)
+* Closed #76 by adding probabilistic comparison methods for 'mdate' objects (e.g. `%l%`, `%le%`, `%g%`, `%ge%`, `%><%`, `%>=<%`)
+* Closed #77 by adding basic vector methods for subsetting and data frames (thanks @WerthPADOH)
+* Added alias function `mdate()` for `as_messydate()`
+* Renamed set family of functions to work as operators (i.e. `%intersect%` and `%union%`)
+* Replaced `is_element()` by `is_subset()` for clarity and consistency
+* Closed #80 by updating `make_messydates()` function to construct ranges of dates
+* Added `impute()` and `to_pool()` functions to help with imputation of messydates for modelling
```

Comment:
I'm not sure whether these two last functions (especially `to_pool()`) are ready yet. Can we remove them and reinstate them to the develop branch after merging/submitting to CRAN?